protocol/test/utils/time.ts

```diff
@@ -0,0 +1,59 @@
+import { ethers } from 'hardhat';
```

Comment:
`helpers.ts` already has a lot of these helper functions. please use those

Reply:
ok